README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for homework 1
+# __hw01 Biography__
+
+## **Degrees**
```

Comment:
bold + heading: keep in mind that because headings are bold by default, emboldening the heading doesn't actually change the appearance of the heading on the final webpage (at least this was my experience)